Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsHand.cs

```diff
@@ -338,13 +349,15 @@ public void UpdateHand(Leap.Hand hand)
                 // Reduce force of hand as it gets further from the original data hand
                 !_ghosted && !_isGrasping && _graspingDeltaCurrent > 0 ? Mathf.InverseLerp(_physicsProvider.HandTeleportDistance * 0.5f, _physicsProvider.HandTeleportDistance, DistanceFromDataHand).EaseOut() : 0f);
 
+            bool bonesOverrotated = AreBonesRotatedBeyondThreshold();
+
             // Fix the hand if it gets into a bad situation by teleporting and holding in place until its bad velocities disappear
-            HandleTeleportingHands();
+            HandleTeleportingHands(bonesOverrotated);
```

Comment:
So you teleport the hands that are in a bad state to somewhere (where they can't be seen?) and where they are not in contact until they restore to a good state?

Reply:
If I understand correctly, 'Teleporting' in this context is teleporting them to the tracked hands location (probably not very far)... but the act of teleporting causes the physics (not visual) hands to be disabled until they are safe to re-enable... meaning they will go through objects until they can safely be re-enabled

Reply:
That makes sense. Thanks